docs/basics.rst

```diff
@@ -607,8 +607,27 @@ __ http://pypi.python.org/pypi/htmllaundry/
                                      preparer=htmllaundry.sanitize,
                                      validator=colander.Length(1))
 
+If you want to create a subclass of :class:`~colander.SchemaNode` and define
+``preparer`` as its class variable, you'll need to wrap your preparer in
```

Comment:
```rst
:class:`~colander.SchemaNode`
```
renders as [SchemaNode](http://docs.pylonsproject.org/projects/colander/en/latest/api.html#colander.SchemaNode)

```rst
:class:`colander.SchemaNode`
```

renders as [colander.SchemaNode](http://docs.pylonsproject.org/projects/colander/en/latest/api.html#colander.SchemaNode)

I think the latter is more appropriate in this context.

The other changes are good.

Ping @mmerickel for technical review.